src/OrchardCore.Modules/OrchardCore.Security/SecurityHeaderDefaults.cs

```diff
@@ -58,9 +60,9 @@ public static class SecurityHeaderDefaults
 
     public static readonly string[] ContentSecurityPolicy = [];
 
-    public static readonly string ContentTypeOptions = ContentTypeOptionsValue.NoSniff;
+    public const string ContentTypeOptions = ContentTypeOptionsValue.NoSniff;
```

Comment:
We don't need to covert all the `static readonly` fields into a constants

Reply:
Const fields should be used when possible for better performance. Cont fields are computed during build time and stored in the metadata. Static fields are computed during runtime. Also the compiler has to create a static constructor in every class that has a static field to be able to initialize the static field at runtime.

Reply:
I know what you mean, check `String.Empty` for instance https://referencesource.microsoft.com/#mscorlib/system/string.cs,c9f70a27facb27cf

Reply:
Okay. Anything else?

Reply:
For that, I suggest reverting changes in some places, constants when the value will never change by it's nature IMHO

Reply:
I know there's a performance benefit, but why `String.Empty` is not considered as `const`?

Reply:
`String.Empty` is a bad example. That one is special-cased by the JIT. In our case we want to avoid having static c'tor if possible.

Reply:
@hishamco here is what Microsoft commented on `string.Empty` 

```
// The Empty constant holds the empty string value. It is initialized by the EE during startup.
// It is treated as intrinsic by the JIT as so the static constructor would never run.
// Leaving it uninitialized would confuse debuggers.
```

Avoiding static constructors when possible is a good approach when possible.